source/runbooks/revoke-network-access.html.md.erb

```diff
@@ -0,0 +1,174 @@
+---
+owner_slack: "#modernisation-platform"
+title: Revoke network access if required
+last_reviewed_on: 2024-03-18
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+# NOTE
+
+Before commencing any work on stopping network access contact the application team to ensure they know what is happening. If there are specific issues they are aware of it may be they want to undertake some of this work themselves or they have another option available.
+
+If the work is required outside normal hours then the teams are unlikely to respond and you can continue with the items listed below. Ensure that the teams are aware before starting this if it happens inside working hours.
+
+# Summary
+
+The steps listed deny access on a platform level. If additional changes are required at an application level then these must be undertaken by the application owning team. 
+
+A network attack is an exceptional situation so quick amendments are needed. It is anticipated that this can be done quickly through the console.
+
+This document lists the steps needed to stop access to and from the internet and preventing the system from being accessed from or accessing other applications on the MoJ.
```

Comment:
This document lists the steps required to revoke internet access, and isolate the compromised infrastructure from other applications.

Maybe?